docs/user-guide/src/capm3/introduction.md

```diff
@@ -0,0 +1,81 @@
+# Kubernetes Cluster API Provider Metal3
+
+Kubernetes-native declarative infrastructure for Metal3.
+
+<div>
+    <img src="../images/metal3-color.svg" width="120px" />
+</div>
+
+## What is the Cluster API Provider Metal3
+
+The [Cluster API](https://github.com/kubernetes-sigs/cluster-api/) brings declarative,
+Kubernetes-style APIs to cluster creation, configuration and management. The API
+itself is shared across multiple cloud providers. Cluster API Provider Metal3 is
+one of the providers for Cluster API and enables users to deploy a Cluster API based
+cluster on top of bare metal infrastructure using Metal3.
+
+## Compatibility with Cluster API
+
+| CAPM3 version | Cluster API version | CAPM3 Release |
+|---------------|---------------------|---------------|
+| ~~v1alpha4~~  | ~~v1alpha3~~        | ~~v0.4.X~~    |
+| v1alpha5      | v1alpha4            | v0.5.X        |
+| v1beta1       | v1beta1             | v1.1.X        |
+
+## Development Environment
+
+There are multiple ways to setup a development environment:
+
+- [Using Tilt](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#tilt-development-environment)
+- [Other management cluster](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#development-using-Kind-or-Minikube)
+- See [metal3-dev-env](https://github.com/metal3-io/metal3-dev-env) for an
+  end-to-end development and test environment for
+  `cluster-api-provider-metal3` and
+  [baremetal-operator](https://github.com/metal3-io/baremetal-operator).
+
+## Getting involved and contributing
+
+Are you interested in contributing to Cluster API Provider Metal3? We, the maintainers and community, would love your suggestions, contributions, and help! Also, the maintainers can be contacted at any time to learn more about how to get involved.
+
+To set up your environment checkout the [development environment](#development-environment).
+
+In the interest of getting more new people involved, we tag issues with [good first issue](https://github.com/metal3-io/cluster-api-provider-metal3/labels/good%20first%20issue). These are typically issues that have smaller scope but are good ways to start to get acquainted with the codebase.
+
+We also encourage ALL active community participants to act as if they are maintainers, even if you don’t have "official" write permissions. This is a community effort, we are here to serve the Kubernetes community. If you have an active interest and you want to get involved, you have real power! Don’t assume that the only people who can get things done around here are the "maintainers".
+
+We also would love to add more “official” maintainers, so show us what you can do!
+
+All the repositories in the Metal3 project, including the Cluster API Provider Metal3 GitHub repository, use the Kubernetes bots. See a full list of the commands [here.](https://go.k8s.io/bot-commands) Note that some of them might not be implemented in metal3 CI.
```

Comment:
Nit: We do not use K8s bot, if you meant - https://github.com/k8s-ci-robot, we use [metal3-io-bot](https://github.com/metal3-io-bot) which is a custom bot for Metal3

Reply:
I mean Kubernetes prow bot commands here which we use in our CI.

Reply:
When it is read, sounds like we are saying we use k8s bot, maybe you can make it a bit clearer, i.e:
```suggestion
All the repositories in the Metal3 project, including the Cluster API Provider Metal3 GitHub repository, use the Kubernetes bot commands and the full list of the commands can be found [here.](https://go.k8s.io/bot-commands) Note that some of them might not be implemented in metal3 CI.
```

Reply:
@furkatgofurov7 I am a bit confused, you said we are not using https://github.com/k8s-ci-robot, but we do right since prow comes from https://github.com/k8s-ci-robot/test-infra. And metal3-io-bot is a github bot which acts on this commands. Am I missing something?

Reply:
@kashifest prow comes from test infra yes, but the bot can be different to interact with the prow cluster, for us it is metal3-io-bot that interacts with it, while i.e CAPI uses native test infra bot since all the configurations for their prow cluster is there.  
Here is https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/Legacy%20Snapshot/prow/README.md#prow-in-the-wild list of other projects that uses prow but if you check carefully, each of them have their own custom bot to interact with their own prow cluster.